third_party/go9p/unpack.go

```diff
@@ -218,7 +218,7 @@ func Unpack(buf []byte, dotu bool) (fc *Fcall, err error, fcsz int) {
 		goto szerror
 	}
 
-	return
+	return //NOSONAR
 
 szerror:
 	return nil, &Error{"invalid size", EINVAL}, 0
```

Comment:
Instead of using `NOSONAR`, could you please refactor the code so that it returns the error inside the `szerror` label and remove the usage of Goto completely? I think that would be much better.

Reply:
The check is buggy, since the use of `goto` makes it entirely possible to have two exit points.